### ISSUE TITLE
Ignoring app also disable tiling for that app.

### ISSUE THREAD

**Issue by bric3** [rxhanson/Rectangle#1469]
I'm ignoring an app via the Rectangle menu because it has conflicting shortcuts. However this also disable tiling for this app.

I remember in the past Rectangle used to only "disable shortcuts" when ignoring an app. I believe this might be a regression. Or is there a new setting somewhere to keep tiling when ignoring an app ?

<img src="https://github.com/user-attachments/assets/dabf7096-6de4-4e78-9182-1e610310e95a" width="300" />

_By the way I had this issue with macOs 14.x as well._

----
macOS version: 15
Rectangle version: 0.83 (89)
Logs if applicable (In Rectangle menu, hold option, "View Logging..."):

**Comment by rxhanson** [comment]
This change was made in v0.70, and you can get back to how it was done prior to v0.70 with the following Terminal command:

```
defaults write com.knollsoft.Rectangle ignoreDragSnapToo -int 2
```

Then restart the app.

**Comment by bric3** [comment]
Thank you I'll try that. What are the meaning of these values?
Maybe it's mentioned somewhere. (I never used the rectangle cli before)

**Comment by rxhanson** [comment]
`defaults` is the preferences storage system in macOS
`com.knollsoft.Rectangle` is the identifier for Rectangle
`ignoreDragSnapToo` is the name I gave this preference
`2` is a value that, for this preference, means turn it off

**Comment by bric3** [comment]
OK, thanks for the explanation, is it related to this function in swift ?

https://github.com/rxhanson/Rectangle/blob/b841afb185c888072678331f83d242b028bce927/Rectangle/Defaults.swift#L254-L261
 
Also thank disabling this fixed my issue.

**Comment by rxhanson** [comment]
Yep, that's it. Number defaults will be 0 if not set, so adding the extra case of 2 effectively gives us the ability to know if the user has actively disabled something instead of just not performed any configuration.

**Comment by bric3** [comment]
Thanks, I believe the issue is fixed for me, but I still wonder if this wouldn't deserve a UI toggle.
But that's up to you :)

**Comment by alemarengo** [comment]
I have the same issue while using Adobe Indesign that it keeps moving until Rectangle is active on this app too; when disable with "Ignore Indesign" the issue stops. But I have no shortcut available; even RightZoom doesn't work as backup. Any suggestion?

**Comment by rxhanson** [comment]
This is an issue with Adobe Indesign. This was first seen maybe around a year ago, and there was an issue filed with Adobe but I don't know the status of it.
